src/app/filecoin-plus/allocators/utils/getAllocatorsWithDatacap.ts

```diff
@@ -0,0 +1,21 @@
+import { getAllocators } from '../utils/getAllocators'
+import { getDatacapStats } from '../utils/getDatacapStats'
+
+export async function getAllocatorsWithDatacap() {
+  const allocators = await getAllocators()
+  const datacapStats = await getDatacapStats()
```

Comment:
This change runs both API calls in parallel using `Promise.all()` rather than sequentially. Running them in parallel is more efficient since we don't need to wait for `getAllocators()` to finish before starting `getDatacapStats()`. This can significantly reduce the total execution time, especially when the API calls are slow.

```suggestion
  const [allocators, datacapStats] = await Promise.all([
    getAllocators(),
    getDatacapStats()
  ]);
```

Reply:
Done.